en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -36,23 +40,113 @@ often working alongside Core Committers.
 
 CMS Squad members have write access to core repositories in order to work effectively with GitHub issues. They are expected to use those permissions with good judgement for merging pull requests.
 
+### Peer Reviewers
+
+[info]
+This is a new role, and is currently in a trial period. It is likely to change over time as we learn what works and what doesn't.
+[/info]
+
+This role is made up of community members who have demonstrated a willingness to improve Silverstripe CMS through their contributions to the open source supported modules. It empowers community members to have a more active role in ensuring pull requests get merged, and builds more momentum for community-created pull requests.
```

Comment:
> I think "ability" here is better.

Can you please explain your reasoning? I think "willingness" here is better because that's the main attribute that this paragraph is talking about people having. "ability" is discussed in the section about onboarding.

Reply:
Done